server/templates/home/searchresult.html

```diff
@@ -0,0 +1,330 @@
+                  
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2022-2023</span> 
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2021-2022</span> 
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2020-2021</span> 
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                </ul>
+              </div>
+
+              <div class="dropdown">
+                <button class="custom-dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+                    <h6>Language</h6>  <i class="fas fa-caret-down"></i>
+                </button>
+                <ul class="dropdown-menu">
+                  
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2022-2023</span> <span id="count2">112</span>
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2021-2022</span> <span id="count2">82</span>
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                  <li>
+                    <div class="check">
+                    <label class="form-check-label" for="flexCheckDefault">
+                      <span>2020-2021</span> <span id="count2">63</span>
+                    </label>
+                    <input class="form-check-input" type="checkbox" value="" id="flexCheckDefault">
+                  </div>
+                  </li>
+
+                </ul>
+              </div>
+
+              <div class="dropdown">
+                <button class="custom-dropdown-toggle" type="button" data-bs-toggle="dropdown" aria-expanded="false">
+                    <h6>Sub-themes</h6>  <i  class="fas fa-caret-down"></i>
```

Comment:
Same for this and formats, create an array of all the values and pass it in through the context, you can loop through ot here